lib/toMatchEyesScreenshot.js

```diff
@@ -5,7 +5,7 @@ require('dotenv').config();
 module.exports = (options = {}) =>
   async function toMatchScreenshot(
     instance,
-    {key, version = 'v1.0.0', viewport, matchLevel},
+    {key, version = 'v1.0.0', viewport, matchLevel, autoSaveNewTest = true},
```

Comment:
This is a breaking change, no?
If so, do we prefer to have a default value of `false`, or bump the lib version?
@yanivefraim what do you say?

Other than that looks great 👍

Reply:
My mistake, it was supposed to be default set to false, so it won't be a breaking change.

Reply:
This is not a breaking change because this is the default value by eyes anyway. So if the value is true, everything remains the same.

Reply:
oh, right, my mistake